NEWS.md

```diff
@@ -1,5 +1,102 @@
 MXNet Change Log
 ================
+
+## 1.3.1
+
+### Bug fixes
+
+* [MXNET-953] Fix oob memory read (v1.3.x) / [#13118](https://github.com/apache/incubator-mxnet/pull/13118)  
+Simple bugfix addressing an out-of-bounds memory read.
+
+
+* [MXNET-969] Fix buffer overflow in RNNOp (v1.3.x) / [#13119](https://github.com/apache/incubator-mxnet/pull/13119)  
+This fixes an buffer overflow detected by ASAN.
```

Comment:
nitpick: ~an~
should be "This fixes a buffer overflow detected" right?

Reply:
Yes, should have been catched [here](https://github.com/apache/incubator-mxnet/pull/12603#issue-216778203) already